test/ILLink.Tasks.IntegrationTests/IntegrationTestBase.cs

```diff
@@ -58,23 +55,21 @@ protected void AddLinkerReference(string csproj)
 		{
 			var xdoc = XDocument.Load(csproj);
 			var ns = xdoc.Root.GetDefaultNamespace();
-			bool added = false;
-			foreach (var el in xdoc.Root.Elements(ns + "ItemGroup")) {
-				if (el.Elements(ns + "PackageReference").Any()) {
-					el.Add(new XElement(ns+"PackageReference",
-						new XAttribute("Include", TestContext.TasksPackageName),
-						new XAttribute("Version", TestContext.TasksPackageVersion)));
-					added = true;
-					break;
-				}
-			}
-			if (!added) {
-				xdoc.Root.Add(new XElement(ns + "ItemGroup",
-					new XElement(ns + "PackageReference",
-						new XAttribute("Include", TestContext.TasksPackageName),
-						new XAttribute("Version", TestContext.TasksPackageVersion))));
+
+			foreach (var el in xdoc.Root.Elements(ns + "PropertyGroup")) {
+				el.Add(new XElement(ns + "ILLinkTasksAssembly", TestContext.TasksAssemblyPath));
+				break;
 			}
 
+			// Workaround to avoid passing an argument that was removed (-l).
```

Comment:
Could you please create issue about this work item

Reply:
Created https://github.com/mono/linker/issues/979.